packages/e/eopkg4-bin/package.yml

```diff
@@ -0,0 +1,30 @@
+name       : eopkg4-bin
+version    : 4.0.0
+release    : 1
+source     :
+    - git|https://github.com/getsolus/eopkg : 38e98ba789d124bec649141ee569bb472832d813
+homepage   : https://github.com/getsolus/eopkg
+license    : GPL-2.0-or-later
+component  : system.base
+summary    : "Python3 port of eopkg/pisi. WARNING: For testing only."
```

Comment:
quotes again

Reply:
This one has quotes on purpose. The : in the summary freaks out YAML.

Reply:
as you were then